docs/source/release/v3.8.0/framework.rst

```diff
@@ -49,6 +49,10 @@ Improved
 - :ref:`SetSample <algm-SetSample>`: Fixed a bug with interpreting the `Center` attribute for cylinders/annuli 
 
 
+- :ref:`RenameWorkspace <algm-RenameWorkspace>` and `RenameWorkspaces <algm-RenameWorkspaces>`
+  now check if a Workspace with that name already exists in the ADS and gives
+  the option to override it.
```

Comment:
override => overwrite? If you modify this we don't need to wait for the builds again.